website/docs/d/cloud_project_database_users.html.markdown

```diff
@@ -29,13 +29,7 @@ output "user_ids" {
 * `service_name` - (Required) The id of the public cloud project. If omitted,
   the `OVH_CLOUD_PROJECT_SERVICE` environment variable is used.
 
-* `engine` - (Required) The engine of the database cluster you want to list users. To get a full list of available engine visit:
```

Comment:
Why are you removing the list of available engines? :)

Reply:
I re-add full list doc, that was a mistake to remove it). 
But remove `Available engines`list because all engines are available for this datasource.